### ISSUE TITLE
ZFIN genes created in September 2019 and later are not available in Noctua (but available in our GPI)

### ISSUE THREAD

**Issue by sabrinatoro** [geneontology/helpdesk#249]
I want to create an annotation to linc.terminator (ZFIN:ZDB-LINCRNAG-190911-1), but this gene is not available in Noctua. I am using the new Noctua form, but the issue is the same in the graph editor. 
1) I am unable to create an annotation using the ID alone - I don't know if this is an expected behavior. 
2) It looks like the new genes created in September 2019 and later are not available in Noctua. 
(examples: ZFIN:ZDB-LINCRNAG-190911-1, ZFIN:ZDB-GENE-190924-2, ZFIN:ZDB-GENE-200114-3). These genes are available in our GPI files. 
This makes me think that there has been a problem with our GPI files since September 2019: 
- either GOC has not been retrieving our latest GPI since September and/or there has been a problem to add the information in Noctua
- or there is a problem with our GPI files and they create an error in GO/Noctua (we have not been notified about such issue, but it is a possibility it is happening). 
Could you please look into this? Thank you

**Comment by suzialeksander** [comment]
@sabrinatoro I'm fairly certain there's a ticket about most of this, let me see if I can find it

**Comment by suzialeksander** [comment]
I may be thinking of an email with @vanaukenk about SGD names not popping up in Noctua: "WB has similar delay issues with Protein2GO in that nomenclature updates that WB makes may take a while to trickle down to UniProtKB and thus Protein2GO. One immediate solution may be for SGD to produce a gpi file using the 1.2 specs. GO could then pick that file up and use it to populate entries in NEO with the most up-to-date id and nomenclature. We do that for WB right now, i.e. we produce a GAF for annotation and a gpi for NEO.

@sabrinatoro It looks like you do produce a gpi according to your yaml. Are you absolutely certain the "fail" cases you listed are in the gpi provided to GO? I don't see 190911-1, 190924-2 or 200114-3 in the file I just obtained at http://current.geneontology.org/annotations/zfin.gpi.gz

also slightly similar https://github.com/geneontology/noctua/issues/583 https://github.com/geneontology/neo/issues/36

@kltm is there a better home for this issue than helpdesk? Not sure it's exactly a Noctua issue

**Comment by sabrinatoro** [comment]
@suzialeksander 
yes, these genes are in the latest gpi file we produced at ZFIN. 
What I do not know is whether this new gpi file is available at GO (i.e. maybe there is a problem with GO getting the file? or maybe there are some error with it).
